types/shared.d.ts

```diff
@@ -85,5 +85,5 @@ export interface LocaleSpecificConfig<ThemeConfig = any> {
 
 export type LocaleConfig<ThemeConfig = any> = Record<
   string,
-  LocaleSpecificConfig<ThemeConfig> & { label: string; link?: string }
+  LocaleSpecificConfig<ThemeConfig> & { label: string; link?: string, languageButtonAriaLabel?: string }
```

Comment:
Add this directly in ThemeConfig.

Reply:
Sorry, not sure if I understand you correctly. Are you referring to `types/default-theme.d.ts`?

If I understand your request correctly, you'd like to add this property for `themeConfig`. I think it's better to keep it as a _locale specific_ property, since labels should be different for every language, preferably.

Reply:
> Are you referring to types/default-theme.d.ts? If I understand your request correctly, you'd like to add this property for themeConfig. 

Yeah. 

> I think it's better to keep it as a locale specific property, since labels should be different for every language, preferably.

`themeConfig` is overridable inside locales. Refer the other labels that are present in that file. Things should not be kept at top level unless it's theme independent, which in your case is not true (Themes other than default theme might not respect that value at all.)

Reply:
Ok. I thought it might not be suitable for `themeConfig`, since the settings would only be relevant if one has a `locales` configuration. Without other languages, the configuration has no effect.

I've updated the PR accordingly. Please have a look and let me know if I got everything right. Thanks!

Reply:
Discard the changes in shared.d.ts, langs.ts, i18n.md. And update VPNavBarTranslations.vue to directly use `theme.langMenuAriaLabel`

Reply:
Thanks! I've updated the commits.

I wonder if one can still change the `aria-label` for different locales this way. But if I understood you correctly, it should be possible nevertheless.

Hope the default value is ought to be set like that.

Reply:
Yeah one can override them:

```ts
import { defineConfig } from 'vitepress'

export default defineConfig({
  // shared properties and other top-level stuff...

  locales: {
    root: {
      label: 'English',
      lang: 'en',

      themeConfig: {
          // override labels here for english
      }
    },
    fr: {
      label: 'French',
      lang: 'fr', // optional, will be added  as `lang` attribute on `html` tag
      link: '/fr/guide', // default /fr/ -- shows on navbar translations menu, can be external

      // other locale specific properties...
      themeConfig: {
          // override labels here for french
      }
    }
  }
})

```

(We're working on updating the docs. Don't want conflicts there that's why leave that i18n.md unchanged for now.)

Reply:
Yeah, I've seen it in the docs anyway. Just wasn't sure if there's something else that needs to be updated. Thanks!